### ISSUE TITLE
ERROR: failed to do request: Head dial tcp: lookup registry- 1.docker.io: no such host, failed to solve with frontend dockerfile.v0: failed to create LLB definition

### ISSUE THREAD

**Issue by evandrocoan** [moby/moby#42867]
**Description**

I am not sure why this happens. But sometimes one of my docker build fails with:
```
Untagged: deploy_image:latest
Deleted: sha256:c54236c653a3178e6debc8ffbffe4d2eba4f789ee236089032b9d2feaa0daa66

#1 [internal] load build definition from deploy.Dockerfile
#1 sha256:9048cafd6715b4e09a204545a8d8ef1cef6209684ecd7e3b69a2e38510a20e81 
#1 transferring dockerfile: 45B done
#1 DONE 0.1s

#2 [internal] load .dockerignore                                                                                            
#2 sha256:ef4db4ddb70d86cffc65cd1e615cd739f059f06ea9b7b98a3f4cc38cc834b527                                                  
#2 transferring context: 34B done                                                                                           
#2 DONE 0.1s                                                                                                                
                                                                                                                            
#3 [internal] load metadata for docker.io/library/debian:11                                                                 
#3 sha256:b037bde4559b1b41e623e1a26f1831a526e95f96feca7ab049a8fc9144d77cd7                                                  
#3 ERROR: failed to do request: Head https://registry-1.docker.io/v2/library/debian/manifests/11: dial tcp: lookup registry-
1.docker.io: no such host                                                                                                   
------                                                                                                                      
 > [internal] load metadata for docker.io/library/debian:11:                                                                
------                                                                                                                      
failed to solve with frontend dockerfile.v0: failed to create LLB definition: failed to do request: Head https://registry-1.
docker.io/v2/library/debian/manifests/11: dial tcp: lookup registry-1.docker.io: no such host 
```

And I just run the build process again, and the error stops.

Perhaps this is related to the group of commands I use. I have another docker build on the same pipeline which does not seem to fail with this error. The only difference between them is the usage of `RUN --mount`. The one that uses `RUN --mount` fails this the error once in a while.

**Steps to reproduce the issue:**
1. Use an docker file without a run with `RUN --mount` and another with it (for example https://github.com/moby/moby/issues/42864).
```dockerfile
FROM debian:11
WORKDIR /
RUN --mount=type=cache,target=/cache/ /bin/bash -c "set -ex;"
```
1. Create the `run.sh` bash file with the following contents and run it. This is the pipeline I use and where the problem happens.
    ```bash
    # Remove the old image while we have a reference to it
    docker rmi build_image:latest || cd .;
    
    DOCKER_BUILDKIT=1 docker build \
        --progress=plain \
        --tag build_image \
        --file build.Dockerfile \
        .
    
    docker run --rm \
        --volume root/:/root \
        build_image \
        /root/build.sh \
        .
    
    docker rmi deploy_image:latest || cd .;
    
    DOCKER_BUILDKIT=1 docker build \
        --progress=plain \
        --tag deploy_image \
        --file deploy.Dockerfile \
        .
    
    # Clean up dangling build caches from the buildkit last build cache
    # https://github.com/moby/buildkit/issues/1359
    docker builder prune --force --filter type=regular;
    ```
1. After running the `run.sh` several times, one of them is going to on the `deploy_image` creation which uses the `RUN --mount`. The first docker build with the `build_image` creation (which does not have a `RUN --mount`) does not failed so far. 

**Describe the results you received:**
Docker image build randomly fails.

**Describe the results you expected:**
Docker image build never failing randomly because it cannot connect to the docker tcp daemon?

**Output of `docker version`:**
```
$ docker version
Client: Docker Engine - Community
 Version:           19.03.13
 API version:       1.40
 Go version:        go1.13.15
 Git commit:        4484c46d9d
 Built:             Wed Sep 16 17:02:52 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          20.10.2
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       8891c58
  Built:            Mon Dec 28 16:15:19 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.3
  GitCommit:        269548fa27e0089a8b8278fc4fc781d7f65a939b
 runc:
  Version:          1.0.0-rc92
  GitCommit:        ff819c7e9184c13b7c2607fe6c30ae19403a7aff
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**Output of `docker info`:**

```
$ docker info
Client:
 Debug Mode: false

Server:
 Containers: 1
  Running: 1
  Paused: 0
  Stopped: 0
 Images: 6
 Server Version: 20.10.2
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 269548fa27e0089a8b8278fc4fc781d7f65a939b
 runc version: ff819c7e9184c13b7c2607fe6c30ae19403a7aff
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-47-generic
 Operating System: Ubuntu 20.04.1 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 15.52GiB
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
WARNING: No blkio weight support
WARNING: No blkio weight_device support
```

May be related to: https://github.com/moby/moby/issues/18842

**Comment by thaJeztah** [comment]
> May be related to https://github.com/moby/moby/issues/18842

Hm... hard to say; the error is a DNS lookup failure, for which there can be many causes.

I do see you're running an outdated version of docker and containerd/runc (as well as a non-matching version of the cli). To exclude the chance it's fixed in more patch versions, would you be able to test if it's still an issue on the latest versions? The cli, engine, and containerd are in separate packages, so to update all of them, you can run something like;

```bash
apt-get update && apt-get install docker-ce docker-ce-cli containerd.io
```

**Comment by evandrocoan** [comment]
Would this issue be related/duplicate to this other fixed issue: https://github.com/moby/buildkit/issues/1958 ?

I updated things, these my versions now.  How can I see the BuildKit version I am using? I tried finding it on this output but I could not figure out if it is here.
<details>
<p>

```
$ sudo apt-get update && sudo apt-get install docker-ce docker-ce-cli containerd.io
Get:1 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Hit:2 http://archive.canonical.com/ubuntu focal InRelease
Hit:3 http://br.archive.ubuntu.com/ubuntu focal InRelease
Get:4 http://dl.google.com/linux/chrome/deb stable InRelease [1.811 B]
Get:5 http://br.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Hit:6 http://us.archive.ubuntu.com/ubuntu focal InRelease
Hit:7 http://linux.teamviewer.com/deb stable InRelease
Get:8 http://us.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Hit:9 http://ppa.launchpad.net/wireshark-dev/stable/ubuntu focal InRelease
Get:10 http://br.archive.ubuntu.com/ubuntu focal-updates/universe Sources [188 kB]
Get:11 http://packages.microsoft.com/repos/code stable InRelease [10,4 kB]
Get:12 http://dl.google.com/linux/chrome/deb stable/main amd64 Packages [1.099 B]
Hit:13 https://repo.fortinet.com/repo/6.4/ubuntu /bionic InRelease
Hit:14 https://download.sublimetext.com apt/stable/ InRelease
Hit:15 https://download.docker.com/linux/ubuntu focal InRelease
Get:16 http://packages.microsoft.com/repos/code stable/main armhf Packages [47,3 kB]
Get:17 http://security.ubuntu.com/ubuntu focal-security/main Sources [178 kB]
Get:18 http://us.archive.ubuntu.com/ubuntu focal-backports InRelease [101 kB]
Get:19 http://us.archive.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Get:20 http://packages.microsoft.com/repos/code stable/main amd64 Packages [46,4 kB]
Get:21 http://packages.microsoft.com/repos/code stable/main arm64 Packages [47,4 kB]
Get:22 http://us.archive.ubuntu.com/ubuntu focal-updates/main Sources [439 kB]
Get:23 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [1.214 kB]
Get:24 http://us.archive.ubuntu.com/ubuntu focal-updates/main i386 Packages [536 kB]
Get:25 http://us.archive.ubuntu.com/ubuntu focal-updates/main Translation-en [260 kB]
Get:26 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 DEP-11 Metadata [283 kB]
Get:27 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 c-n-f Metadata [14,3 kB]
Get:28 http://us.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 DEP-11 Metadata [940 B]
Get:29 http://us.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [855 kB]
Get:30 http://us.archive.ubuntu.com/ubuntu focal-updates/universe i386 Packages [633 kB]
Get:31 http://us.archive.ubuntu.com/ubuntu focal-updates/universe amd64 DEP-11 Metadata [352 kB]
Get:32 http://us.archive.ubuntu.com/ubuntu focal-updates/universe amd64 c-n-f Metadata [18,9 kB]
Get:33 http://us.archive.ubuntu.com/ubuntu focal-backports/universe amd64 DEP-11 Metadata [10,4 kB]
Get:34 http://us.archive.ubuntu.com/ubuntu focal-security/main amd64 Packages [867 kB]
Get:35 http://us.archive.ubuntu.com/ubuntu focal-security/main amd64 DEP-11 Metadata [27,7 kB]
Get:36 http://us.archive.ubuntu.com/ubuntu focal-security/universe amd64 DEP-11 Metadata [61,0 kB]
Get:37 http://us.archive.ubuntu.com/ubuntu focal-security/multiverse amd64 DEP-11 Metadata [2.464 B]
Fetched 6.651 kB in 3s (2.045 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  docker-scan-plugin
The following NEW packages will be installed:
  docker-scan-plugin
The following packages will be upgraded:
  containerd.io docker-ce docker-ce-cli
3 upgraded, 1 newly installed, 0 to remove and 661 not upgraded.
Need to get 88,7 MB of archives.
After this operation, 52,5 MB disk space will be freed.
Do you want to continue? [Y/n] y
Get:1 https://download.docker.com/linux/ubuntu focal/stable amd64 containerd.io amd64 1.4.9-1 [24,7 MB]
Get:2 https://download.docker.com/linux/ubuntu focal/stable amd64 docker-ce-cli amd64 5:20.10.8~3-0~ubuntu-focal [38,8 MB]
Get:3 https://download.docker.com/linux/ubuntu focal/stable amd64 docker-ce amd64 5:20.10.8~3-0~ubuntu-focal [21,2 MB]
Get:4 https://download.docker.com/linux/ubuntu focal/stable amd64 docker-scan-plugin amd64 0.8.0~ubuntu-focal [3.889 kB]
Fetched 88,7 MB in 21s (4.314 kB/s)
(Reading database ... 342953 files and directories currently installed.)
Preparing to unpack .../containerd.io_1.4.9-1_amd64.deb ...
Unpacking containerd.io (1.4.9-1) over (1.4.3-1) ...
Preparing to unpack .../docker-ce-cli_5%3a20.10.8~3-0~ubuntu-focal_amd64.deb ...
Unpacking docker-ce-cli (5:20.10.8~3-0~ubuntu-focal) over (5:19.03.13~3-0~ubuntu-focal) ...
Preparing to unpack .../docker-ce_5%3a20.10.8~3-0~ubuntu-focal_amd64.deb ...
Unpacking docker-ce (5:20.10.8~3-0~ubuntu-focal) over (5:20.10.2~3-0~ubuntu-focal) ...
Selecting previously unselected package docker-scan-plugin.
Preparing to unpack .../docker-scan-plugin_0.8.0~ubuntu-focal_amd64.deb ...
Unpacking docker-scan-plugin (0.8.0~ubuntu-focal) ...
Setting up docker-scan-plugin (0.8.0~ubuntu-focal) ...
Setting up containerd.io (1.4.9-1) ...
Setting up docker-ce-cli (5:20.10.8~3-0~ubuntu-focal) ...
Setting up docker-ce (5:20.10.8~3-0~ubuntu-focal) ...
Processing triggers for man-db (2.9.1-1) ...
Processing triggers for systemd (245.4-4ubuntu3.10) ...

$ docker version
Client: Docker Engine - Community
 Version:           20.10.8
 API version:       1.41
 Go version:        go1.16.6
 Git commit:        3967b7d
 Built:             Fri Jul 30 19:54:27 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.8
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.6
  Git commit:       75249d8
  Built:            Fri Jul 30 19:52:33 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.9
  GitCommit:        e25210fe30a0a703442421b0f60afac609f950a3
 runc:
  Version:          1.0.1
  GitCommit:        v1.0.1-0-g4144b63
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

$ docker info
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  app: Docker App (Docker Inc., v0.9.1-beta3)
  buildx: Build with BuildKit (Docker Inc., v0.6.1-docker)
  scan: Docker Scan (Docker Inc., v0.8.0)

Server:
 Containers: 2
  Running: 0
  Paused: 0
  Stopped: 2
 Images: 11
 Server Version: 20.10.8
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runtime.v1.linux runc io.containerd.runc.v2
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: e25210fe30a0a703442421b0f60afac609f950a3
 runc version: v1.0.1-0-g4144b63
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: default
 Kernel Version: 5.4.0-47-generic
 Operating System: Ubuntu 20.04.1 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 15.52GiB
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

WARNING: No swap limit support
```
</p>
</details><br>

I will not be using much docker for some time, if the issue happens again I will report back in some time.

**Comment by evandrocoan** [comment]
I forgot to tell, but could `dind` (docker in docker) influence/cause this error? These steps I mentioned where running inside a `dind`  image, i.e.:
```bash
docker run \
    --privileged \
    --name "${container_name}" \
    --volume "$SOURCE_FOLDER":/src \
    --volume /var/run/docker.sock:/var/run/docker.sock \
    -it \
    docker:dind sh -c "
        set -ex
        cd /src
        apk update
        apk add bash git python3 py3-pip
        sh
    "
```
Then here inside the sh shell of the dind I would run the docker build commands/cache mounts, etc, described on the first post.

**Comment by thaJeztah** [comment]
> These steps I mentioned where running inside a dind image, i.e.:

Looks like you're bind-mounting the docker socket into the container, so effectively it's not docker-in-docker (it's just the `docker` cli running in the container, but everything else runs directly on the daemon running on the host (outside the container); If you bind-mount the socket, the cli inside the container connects to the docker daemon API using that socket, so it's directly controlling the daemon on the host.

**Comment by project-administrator** [comment]
Such lookup errors happen on my system **after** I re-connect my host (laptop) to another network that has a different DNS server. E.g. suspend the laptop and wake it up on another wifi network. Docker daemon seemingly tries to use the DNS server which it has recorded during docker service startup, it would not update the DNS settings if the host system DNS settings change suddenly.

**Comment by evandrocoan** [comment]
@project-administrator I think you nailed down the problem. I am connected to a VPN which (which sometimes messes with my DNS) probably goes down sometimes and each time I run docket-compose to run unit tests, my wifi network connection does down and up (I am not sure why yet). But I could not see a direct relation yet, but I will try to pay more attention next time I see the problem (if I see it again, because I just changed my VPN software to another, which my not mess with my DNS).

But this problem fixes itself without restarting the docker daemon. If the `Docker daemon seemingly tries to use the DNS server which it has recorded during docker service startup` wouldn't it require me to restart the docker daemon to fix it?

**Comment by project-administrator** [comment]
@evandrocoan I would speculate that the DNS lookup problem might be fixed without the docker service restart if either:
a) you switch to the previous network configuration that has been active when docker service has been started
b) the DNS server is still accessible in the new network that you've connected to

In my case I switch to another wifi network and it has a different DNS IP and I need to restart the docker service to fix this.

**Comment by okvik** [comment]
FWIW it's happening to me as well when switching between work and home networks.

I've found it's enough to simply restart your builder container to get it working again.

**Comment by bassg0navy** [comment]
I use colima to run builds. Tearing down then restarting colima service/VM and restarting did the trick for me. My builds started working and had no further connectivity issues.

**Comment by andig** [comment]
Had a similar problem (https://community.fly.io/t/deploy-errors-fly-can-t-communicate-with-its-own-docker-hub-mirror-part-2-solved/12439/3):

```
GOOS=linux GOARCH=amd64 go build -o soc-server -tags=release -ldflags='-X github.com/andig/evcc-cloud/main.Commit=35abdff -s -w' github.com/andig/evcc-cloud
docker build --platform linux/amd64 -t andig/evcc-cloud --push .

[+] Building 30.0s (3/3) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                               0.0s
 => => transferring dockerfile: 469B                                                                                                                               0.0s
 => [internal] load .dockerignore                                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                    0.0s
 => ERROR [internal] load metadata for docker.io/library/alpine:3.17                                                                                              30.0s
------
 > [internal] load metadata for docker.io/library/alpine:3.17:
------
Dockerfile:1
--------------------
   1 | >>> FROM alpine:3.17 as builder
   2 |     RUN apk update && apk add --no-cache git ca-certificates tzdata && update-ca-certificates
   3 |
--------------------
ERROR: failed to solve: DeadlineExceeded: DeadlineExceeded: DeadlineExceeded: alpine:3.17: failed to do request: Head "https://registry-1.docker.io/v2/library/alpine/manifests/3.17": dial tcp 52.1.184.176:443: i/o timeout
```

Reproducible failure. After

    docker buildx rm

things started working again. Looking like an issue inside the builder to me.

**Comment by thaJeztah** [comment]
Let me close this ticket for now, as it looks like it went stale.

**Comment by mizhou901101** [comment]
> Had a similar problem (https://community.fly.io/t/deploy-errors-fly-can-t-communicate-with-its-own-docker-hub-mirror-part-2-solved/12439/3):
> 
> ```
> GOOS=linux GOARCH=amd64 go build -o soc-server -tags=release -ldflags='-X github.com/andig/evcc-cloud/main.Commit=35abdff -s -w' github.com/andig/evcc-cloud
> docker build --platform linux/amd64 -t andig/evcc-cloud --push .
> 
> [+] Building 30.0s (3/3) FINISHED
>  => [internal] load build definition from Dockerfile                                                                                                               0.0s
>  => => transferring dockerfile: 469B                                                                                                                               0.0s
>  => [internal] load .dockerignore                                                                                                                                  0.0s
>  => => transferring context: 2B                                                                                                                                    0.0s
>  => ERROR [internal] load metadata for docker.io/library/alpine:3.17                                                                                              30.0s
> ------
>  > [internal] load metadata for docker.io/library/alpine:3.17:
> ------
> Dockerfile:1
> --------------------
>    1 | >>> FROM alpine:3.17 as builder
>    2 |     RUN apk update && apk add --no-cache git ca-certificates tzdata && update-ca-certificates
>    3 |
> --------------------
> ERROR: failed to solve: DeadlineExceeded: DeadlineExceeded: DeadlineExceeded: alpine:3.17: failed to do request: Head "https://registry-1.docker.io/v2/library/alpine/manifests/3.17": dial tcp 52.1.184.176:443: i/o timeout
> ```
> 
> Reproducible failure. After
> 
> ```
> docker buildx rm
> ```
> 
> things started working again. Looking like an issue inside the builder to me.

for me is "docker context rm orbstack -f". everything suddenly works!

**Comment by lexvieira** [comment]
Guys in short, you have a network problem, just disconnect or reset your network connection, I used my phone hotspot and I managed to login on Docker, later I disconnected and I was able to come back to the wifi connection and use the docker pull or docker build normally .

![image](https://github.com/moby/moby/assets/11091181/afe94b83-0b2c-4131-b61a-2d53b141dfd5)
![image](https://github.com/moby/moby/assets/11091181/84c0535c-7f6d-4a07-8b1a-bb429a49559e)

I also included the DNSs 8.8.8.8 and 8.8.4.4, just for the wifi that I was connected.

**Comment by tarikwaleed** [comment]
It was just a network problem . I was connected to a remote machine via RDP and figured out that the machine was not connected to the internet.

**Comment by tarikwaleed** [comment]
It was just a network problem . I was connected to a remote machine via RDP and figured out that the machine was not connected to the internet.

**Comment by contentfree** [comment]
I'm getting similar issues with a local image. Why is Orbstack trying to lookup registry-1.docker.io when my local `docker images` shows the image that I'm trying to use with a `FROM`? How do I get Orbstack to use the image that exists locally instead of trying to reach out to the internet?

**Comment by contentfree** [comment]
BTW, I was able to get around the issue by prepending `DOCKER_BUILDKIT=0` to the second image's build command. So it's apparently a problem with Buildkit. Is there anything Orbstack can do about that?

**Comment by imkunalkumar** [comment]
> Guys in short, you have a network problem, just disconnect or reset your network connection, I used my phone hotspot and I managed to login on Docker, later I disconnected and I was able to come back to the wifi connection and use the docker pull or docker build normally .
> 
> ![image](https://private-user-images.githubusercontent.com/11091181/251372077-afe94b83-0b2c-4131-b61a-2d53b141dfd5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2NDI0NzYsIm5iZiI6MTcwOTY0MjE3NiwicGF0aCI6Ii8xMTA5MTE4MS8yNTEzNzIwNzctYWZlOTRiODMtMGIyYy00MTMxLWI2MWEtMmQ1M2IxNDFkZmQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA1VDEyMzYxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA3YWJlYjliNDMyZTQxMWRjZTc1NDJiNjNkNTk3MTUwYjYyZDRmODg3YWIyODEyOWVmZWZiOGJhMzA5ZGFlMDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lWvNxc51goVJui5qov3WGUc8oLwiJoivyOiKDKlbJ4s) ![image](https://private-user-images.githubusercontent.com/11091181/251372134-84c0535c-7f6d-4a07-8b1a-bb429a49559e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2NDI0NzYsIm5iZiI6MTcwOTY0MjE3NiwicGF0aCI6Ii8xMTA5MTE4MS8yNTEzNzIxMzQtODRjMDUzNWMtN2Y2ZC00YTA3LThiMWEtYmI0MjlhNDk1NTllLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA1VDEyMzYxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMyMGZjYmMzZTk4MzkzYmZlZjExYWExYmUwZmE1ODI0NDQ3MzE2Mzc0MjkzOWM5NjI0Y2U2NWNkZjJjNjgwZDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LY3m_Az8Cx7DTiMxYGM91W678d3n2-7ZBLO6hQF7_Tw)
> 
> I also included the DNSs 8.8.8.8 and 8.8.4.4, just for the wifi that I was connected.

it works, thanks

**Comment by b0rt** [comment]
Uninstalling and Reinstalling Docker Desktop `v4.28.0` on my windows 10 with WSL2 helped me
